client/ayon_maya/plugins/load/load_arnold_standin.py

```diff
@@ -14,12 +15,11 @@
 from ayon_maya.api import plugin
 
 
-def is_sequence(files):
-    sequence = False
-    collections, remainder = clique.assemble(files, minimum_items=1)
+def is_sequence(files: list[str]) -> bool:
+    collections, _remainder = clique.assemble(files)
```

Comment:
```suggestion
    collections, _remainder = clique.assemble(files, patterns=[clique.PATTERNS["frames"]])
```
do we need to check on the frames pattern or?

Reply:
I've tweaked the logic - now it only cares about files of same extension - and if it happens to find multiple frames of that on disk AND it matches in any form to be a collection according to `clique` then we'll assume it's a sequence. That way, we don't accidentally say it's a sequence if there happen to be other files next to it that are....

Anyway, the frames pattern I think is redundantly specific in this scenario - so rather not lock it down further until we face an issue.